src/models/fields/IllPerson.ts

```diff
@@ -0,0 +1,7 @@
+export enum IllPerson {
+    director = 'director',
+    accountant = 'accountant',
+    family = 'family',
+    employee = 'employee',
+    otherPerson = 'otherPerson'
```

Comment:
why did you choose to use otherPerson as the name for the field, rather than “someoneElse”. It matches better with what is represented on the page

Reply:
Just following the prototype, but happy to change if you think it's needed. In fact this would help distinguish between the `illPerson` value "otherPerson", and the field called "otherPerson", so I quite like the suggestion.